### PR TITLE
Prepend the module name to the path.

### PR DESCRIPTION
**Pull request by xorrkaz**

### CODE REVIEW

**Comment by xorrkaz** [comment]
Miroslav, can you test-push this?  The new paths should be /Module:.../.../...

**Comment by miroslavKovacPantheon** [comment]
@xorrkaz Done,
can you please check and confirm? for example [here](https://www.yangcatalog.org/yang-search/yang_tree/Cisco-IOS-XR-ipv4-dhcpd-cfg@2018-10-24#)

**Comment by miroslavKovacPantheon** [comment]
This still does not resolve augmentations right?

File: search/templates/search/yang_tree.html [hunk 1]
```diff
@@ -124,8 +124,14 @@ <h3>{{ title }}</h3>
               },
               {
                 header: '<b>Path</b>',
+                maxWidth: 200,
```
Comment (miroslavKovacPantheon) [inline search/templates/search/yang_tree.html]: 200 seemes a bit much. I would go with 50 maybe and people can resize it if they want check it out on the page.. it is implemented

File: search/templates/search/yang_tree.html [hunk 2]
```diff
@@ -124,8 +124,14 @@ <h3>{{ title }}</h3>
...
                 value: 'path'
               },
+              {
+                header: '<b>Sensor Path</b>',
+                maxWidth: 200,
```
Comment (miroslavKovacPantheon) [inline search/templates/search/yang_tree.html]: same as above

**Comment by miroslavKovacPantheon** [comment]
Looks good to me..

**Pull request merged**